wallet/AndroidManifest.xml

```diff
@@ -157,12 +157,6 @@
             android:configChanges="keyboard|keyboardHidden"
             android:screenOrientation="portrait"
             android:theme="@style/LockScreenActivity.Child.Theme"></activity>
-        <activity
-            android:name="de.schildbach.wallet.ui.RequestCoinsActivity"
```

Comment:
Was used in one of the Context Menu options in the `WalletActivity`. Not needed anymore.